Dynatrace/dynatrace-operator
# Generate OtelC token and store it in extensions secret

## Description

[K8S-10324](https://dt-rnd.atlassian.net/browse/K8S-10324)

Generate OtelC token on extension reconcile and add it to the extension secret.

## How can this be tested?

- Apply dynakube with Prometheus enabled
- Check that `<dynakubeName>-extensions-token` secret contains eec-token and otelc-token data.
